test/testfixture/make_functions.go

```diff
@@ -515,3 +516,35 @@ func makeTrackers(fxt *TestFixture) error {
 	}
 	return nil
 }
+
+func makeQueries(fxt *TestFixture) error {
+	if fxt.info[kindQueries] == nil {
+		return nil
+	}
+	fxt.Queries = make([]*query.Query, fxt.info[kindQueries].numInstances)
+	queryRrepo := query.NewQueryRepository(fxt.db)
+
+	for i := range fxt.Queries {
+		fxt.Queries[i] = &query.Query{
+			Title:   testsupport.CreateRandomValidTestName("query "),
+			Fields:  `{"key": "value"}`,
```

Comment:
I highly doubt that this is a valid query expression and it probably only works because there's no check in the query repo's creation function.